packages/models-library/src/models_library/projects_state.py

```diff
@@ -29,14 +29,47 @@ class DataState(str, Enum):
     OUTDATED = "OUTDATED"
 
 
+@unique
+class ProjectStatus(str, Enum):
+    CLOSED = "CLOSED"
+    CLOSING = "CLOSING"
+    CLONING = "CLONING"
+    EXPORTING = "EXPORTING"
+    OPENING = "OPENING"
+    OPENED = "OPENED"
```

Comment:
would you mind providing more context to these states?

Reply:
hmm not sure what you want:
CLOSED = the project is closed
CLOSING = the project is in the process of being closed
etc etc...

Reply:
I wasn't that clear, I wanted more of a lifecycle kind of explanation. But maybe it is not necessary.